docs/rest_api.md

```diff
@@ -173,6 +169,92 @@ Additionally, the `show_networks` api call may return the channel identifier
 Where documentation specifies that the network driver should choose a
 default channel, the VLAN drivers choose `vlan/native`.
 
+### list_networks
+
+`GET /networks`
+
+List all networks.
+
+Returns a JSON dictionary of dictionaries, indexed by the name of the network with interior dictionaries indexed by project and channel.
+
+The response must contain the following fields:
+
+* "network", the name of a network
+* "driver_id", the id of the network
+* "projects", a list of projects with access to the network or 'None' if network is public 
+
+Example Response:  
+	{
+		"netA": {
+			"driver_id": "101", 
+			"projects": ["qproj-01", qproj-02"]
+			},
+		"netB": {
+			"driver_id": "102", 
+			"projects": None}
+	}
+
+Authorization requirements:
+
+Only admins can list the networks.
```

Comment:
Should probably stick to the formulaic language used everywhere else in the document, i.e:

* Administrative access

It may not seem like a big deal, but with specifications you want to leave little room for interpretation. Using the same phrasing for the same thing helps with that. Not a huge problem in this case, but generall good practice.